include/libjungle/jungle_builder.h

```diff
@@ -44,6 +44,11 @@ class Builder {
          */
         std::string path;
 
+        /**
+         * DBConfig for the DB instance to be created.
+         */
+        const DBConfig* dbConfig;
```

Comment:
It should be `DBConfig dbConfig`, not pointer. The caller (of `init`) life span may not be longer than builder's life span.

Reply:
In the init() function, https://github.com/eBay/Jungle/pull/177/files#diff-e81fa7970a0833b1cb0b4bdbfadfefa84eaee52400bbf5d0b49d47bbe5474e63R121-R124

We already copy the passed in db_config to the dbConfig member of `Builder`, the pointer in this `BuildParams` will have the same life span as `Builder`.

Reply:
Ok, now I get it.